framework/src/utils/InputParameters.C

```diff
@@ -1122,8 +1166,9 @@ InputParameters::checkParamName(const std::string & name) const
 }
 
 bool
-InputParameters::shouldIgnore(const std::string & name)
+InputParameters::shouldIgnore(const std::string & name_in)
 {
+  const auto name = checkForRename(name_in);
   auto it = _params.find(name);
   if (it != _params.end())
     return it->second._ignore;
```

Comment:
there's two get routines below that could use a checkforrename ?

Reply:
They only use internal data. For the incoming `group` argument, I'm not doing anything about renaming parameter groups here.